### ISSUE TITLE
[5.x] `ExtensionException` calls `Exception::__construct()` with wrong arguments

### ISSUE THREAD

**Issue by simonhammes** [Codeception/Codeception#6370]
#### What are you trying to achieve?
I use a library which throws exceptions of type `ExtensionException`.

#### What do you get instead?
```
Exception::__construct(): Argument #2 ($code) must be of type int, null given
```

The error seems to originate here: https://github.com/Codeception/Codeception/blob/eefe1abc60500c0516b85131cbbfbb9a22899db4/src/Codeception/Exception/ExtensionException.php#L19

It should either use `previous:` as a named argument or pass `0` (which is the default value) for the second argument.

### Details

* Codeception version: 5.0.x-dev (I am trying to make the library work with the alpha releases of Codeception 5)
* PHP Version: 8.0
* Operating System: Linux
* Installation type: Composer

**Comment by Naktibalda** [comment]
interesting, that line of code is almost 9 years old.

**Comment by Naktibalda** [comment]
Fixed by #6375
